<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DynamicAutomatonWidget</class>
 <widget class="QWidget" name="DynamicAutomatonWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_name_machine">
     <property name="text">
      <string>Machine 1</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_news_box">
     <item>
      <widget class="QPushButton" name="m_new_exp_btn">
       <property name="text">
        <string>Inserir Exp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_new_grammar_btn">
       <property name="text">
        <string>Inserir GR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_grammar_btn">
       <property name="text">
        <string>Ver GR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_new_machine_btn">
       <property name="text">
        <string>Inserir AF</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="m_machine"/>
   </item>
   <item>
    <widget class="QLabel" name="m_label_op">
     <property name="text">
      <string>Operações</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_operations_box">
     <item>
      <widget class="QPushButton" name="m_negation_btn">
       <property name="text">
        <string>!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_closure_btn">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_transitive_btn">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_optional_btn">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_reverse_btn">
       <property name="text">
        <string>Reverso</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_determinization_btn">
       <property name="text">
        <string>Determ.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_minimization_btn">
       <property name="text">
        <string>Mínimo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_n_sentences">
       <property name="text">
        <string>|Sentenças| = n :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_n_of_sentences">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="m_finitude">
       <property name="text">
        <string>Finitude</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_emptiness">
       <property name="text">
        <string>Ling. Vazia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_membership">
       <property name="text">
        <string>Membro :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_sentence">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="m_label_history">
     <property name="text">
      <string>Histórico</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="m_history">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
